flagsmith/webhooks.py

```diff
@@ -0,0 +1,41 @@
+import hashlib
+import hmac
+from typing import Union
+
+
+def generate_signature(
```

Comment:
I can't see any reason that this function should be part of the public interface for this module?

Reply:
Fair enough, yep, ok, happy to leave it in.